pkg/client/clientset/versioned/typed/volumesnapshot/v1beta1/volumesnapshot.go

```diff
@@ -19,6 +19,7 @@ limitations under the License.
 package v1beta1
 
 import (
+	"context"
```

Comment:
How did you modify these files under pkg/client?  They should be generated automatically.

Reply:
@xing-yang I actually generated it in first patchset version, however the CI was failing with the generated clients. So added the fix in clientset manually and CI was happy. I retried generating the clientset again now and its failing. Am I missing something here?

Reply:
I downloaded your PR and removed the last 2 commits (dated 4/6) but somehow I can still build.  I expect that it would complain about interface changes.

Reply:
@xing-yang we have to remove only last commit (`e351b74`) and try build , sorry for the miscommunication. The commit ( `1961ec2`) is the one which actually remove the clientset changes.

Reply:
Ok, I got rid of all 3 commits related to clientset changes and then I couldn't compile any more.  I tried to generate them automatically but ran into the same problems.

For some reason, the code generator is not using the right lib version to generate those files.

Reply:
@xing-yang or is it expected that, the client generator will fail while there is a breaking change? Also what do you think as the next step here? Thanks!

Reply:
We have already deleted the generated files before running the script.  So it should not be that it is a breaking change.  It should just start fresh when generating them.